webcompat/static/js/lib/flash-message.js

```diff
@@ -32,20 +32,22 @@ var FlashMessageView = Backbone.View.extend({
     var timeout = opts.timeout || 4000;
     var message = opts.message;
 
+    this.$el.addClass("notification-information");
     this.render(message);
     setTimeout(_.bind(this.hide, this), timeout);
   },
   showError: function(data) {
-    this.$el.addClass("is-error");
+    this.$el.addClass("is-error notification-alert");
```

Comment:
This means we have `notification-bar js-flashmessage is-error notification-alert notification-information` classes. Is that expected? That is, are `-alert` and `-information` designed to coexist?